mava/configs/env/cleaner.yaml

```
@@ -0,0 +1,18 @@
+# ---Environment Configs---
+defaults:
+  - _self_
+  - scenario: clean-5x5x5a # [clean-10x10x10a, clean-15x15x15a, clean-20x20x20a, clean-30x30x30a]
```

also gotta add in this option
```suggestion
  - scenario: clean-5x5x5a # [clean-5x5x5a, clean-10x10x10a, clean-15x15x15a, clean-20x20x20a, clean-30x30x30a]
```